types/parent-package-json/parent-package-json-tests.ts

```diff
@@ -0,0 +1,10 @@
+import * as parent from 'parent-package-json';
+
+const pathToParent: string = (parent() as parent.ParentPackage).path;
```

Comment:
Test by using a type check:
```ts
const pathToParent = parent();
if (pathToParent !== false) {
    const path: string = pathToParent.path;
    ... other tests that require `ParentPackage` ...
}
```

Reply:
I have restructured the tests. Thank you for the suggestion, this hadn't occurred to me.